crates/bws/src/render.rs

```diff
@@ -49,6 +50,33 @@ pub(crate) fn serialize_response<T: Serialize + TableSerialize<N>, const N: usiz
             let text = serde_yaml::to_string(&data).unwrap();
             pretty_print("yaml", &text, color);
         }
+        Output::Env => {
+            let mut commented_out = false;
+            let mut text: Vec<String> = data
+                .get_values()
+                .into_iter()
+                .map(|row| {
+                    if row[1]
+                        .chars()
+                        .all(|c| c.is_ascii_alphanumeric() || c == '_')
+                        && !row[1].chars().next().unwrap().is_digit(10)
+                    {
```

Comment:
I think this would look cleaner if we used a `regex`:

```rust
let valid_key_regex = regex::Regex::new("^[a-zA-Z_][a-zA-Z0-9_]*$").unwrap();

let mut commented_out = false;
let mut text: Vec<String> = data
    .get_values()
    .into_iter()
    .map(|row| {
        if valid_key_regex.is_match(&row[1]) {
```

Just have to add the crate in the bws/Cargo.toml:
```diff
uuid = { version = "^1.3.3", features = ["serde"] }
+regex = { version = "1.10.2", features=["std", "perf"], default-features=false }
```

Reply:
Agree. I applied those changes.